logging/google/cloud/logging/handlers/handlers.py

```diff
@@ -117,7 +118,7 @@ def setup_logging(handler, excluded_loggers=EXCLUDED_LOGGER_DEFAULTS,
 
         client = google.cloud.logging.Client()
         handler = CloudLoggingHandler(client)
-        google.cloud.logging.setup_logging(handler)
+        google.cloud.logging.handlers.setup_logging(handler)
```

Comment:
I originally had it in `__init__.py` before in review we moved it to be a client method, forgot to update snippet here.